projects/m4c/README.md

```diff
@@ -1,108 +1,3 @@
 # Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA
 
-This repository contains the code for M4C model from the following paper, released under the MMF:
-
-* R. Hu, A. Singh, T. Darrell, M. Rohrbach, *Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA*. in CVPR, 2020 ([PDF](https://arxiv.org/pdf/1911.06258.pdf))
-```
-@inproceedings{hu2020iterative,
-  title={Iterative Answer Prediction with Pointer-Augmented Multimodal Transformers for TextVQA},
-  author={Hu, Ronghang and Singh, Amanpreet and Darrell, Trevor and Rohrbach, Marcus},
-  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
-  year={2020}
-}
-```
-
```

Comment:
Let's keep the citation here atleast(or if you want it in docs, keep a copy over here as well) for consistency with other projects.